dev-support/test.sh

```diff
@@ -18,7 +18,7 @@ function check_build {
     pushd $ROOT_DIR > /dev/null
 
     local result
-    result=`cargo build --release && cargo test --release --all --no-run && cargo bench --all --no-run && ( cd core/src/storage && cargo build --release )`
```

Comment:
This change invalidates the storage benchmark build

Reply:
It looks like that it's originally wrong when it was introduced.

The intended path is core/benchmark/storage as in dev-support/test-mac.sh, however a typo is made in test.sh
  
https://github.com/Conflux-Chain/conflux-rust/commit/026304dfcfe5c990c754d84b31c4fb0c29db8531#diff-891208896b0019cd1ac7a1a4ad8003ba